docs/EN_US/ConfiguringHPCC/ConfiguringHPCC.xml

```diff
@@ -1552,14 +1610,23 @@ sudo -u hpcc cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/envi
 
           <para>This section describes the Esp - myesp Attributes tab.</para>
 
-          <para><graphic fileref="images/CM-img08-1.jpg"
-          vendor="configmgrSS" /></para>
+          <para>
+            <graphic fileref="images/CM-img08-1.jpg" vendor="configmgrSS" />
+          </para>
 
           <!--MyESP-Include-ESP.t6-XXX12-->
 
-          <para><xi:include href="../XMLGeneration/xml/esp.xsd.mod.xml"
-          xpointer="xpointer(//*[@id='ESP.t6'])"
-          xmlns:xi="http://www.w3.org/2001/XInclude" /></para>
+          <para>
+            <xi:include href="../XMLGeneration/xml/esp.xsd.mod.xml"
+                        xpointer="xpointer(//*[@id='ESP.t6'])"
+                        xmlns:xi="http://www.w3.org/2001/XInclude" />
+          </para>
+
+          <!--***Move to XSD for 2.0***-->
+
+          <para>Persistent HTTP connections can cause backwards compatibility
```

Comment:
I don't see any description of the Persistent Connections feature in. general, or explanation of the parameters maxPersistentRequests and maxPersistentIdleTime, are they in a different file that is not included in this PR?